collection.spec.js

```diff
@@ -1,143 +1,152 @@
-const {Collection} = require('./index');
+const { Collection } = require("./index");
+const _ = require("lodash");
```

Comment:
Can you only import required function from ladash

Reply:
sure